week-2-wide-perspective/w02e02-data-manipulation/src/main.js

```diff
@@ -1,8 +1,26 @@
-import { ordersFakeData } from './orders/orders.fake-data.js'
+import { annualOrderTile } from "./orders/components/AnnualOrderTile.js";
+import { orderTile } from "./orders/components/OrderTile.js";
 
-import './orders/tiles/BestSaleTile.js'
-import './orders/tiles/TotalOrdersTile.js'
-import './orders/tiles/TotalIncomeTile.js'
-import './orders/tiles/AnnualOrdersTile.js'
+import OrdersService from "./orders/orders.service.js";
```

Comment:
to jest singleton. nazwałbym go z małych `ordersService`

Duża litera sugeruje klasę + metody statyczne poniżej. A to nie jest prawda :).